devlib/module/cpuidle.py

```diff
@@ -152,3 +152,9 @@ def disable_all(self, cpu=0):
         for state in self.get_states(cpu):
             state.disable()
 
+    def trace_states(self):
```

Comment:
The name is not indicative of what the method does. Should be renamed to something like `perturb_cpus()`.